src/Sylius/Component/Core/Model/ChannelInterface.php

```diff
@@ -60,4 +60,8 @@ public function setSkippingPaymentStepAllowed(bool $skippingPaymentStepAllowed):
     public function isAccountVerificationRequired(): bool;
 
     public function setAccountVerificationRequired(bool $accountVerificationRequired): void;
+
+    public function getShopBillingData(): ?ShopBillingDataInterface;
```

Comment:
If all fields are not required, we might enforce returning an instance of ShopBillingDataInterface so that there's no need for an additional check for nullability.